examples/mean-reversion.go

```diff
@@ -0,0 +1,246 @@
+package main
+
+import (
+	"fmt"
+	"math"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/alpacahq/alpaca-trade-api-go/alpaca"
+	"github.com/alpacahq/alpaca-trade-api-go/common"
+	"github.com/shopspring/decimal"
+)
+
+type alpacaClientContainer struct {
+	client         *alpaca.Client
+	closingPrices  []float64
+	runningAverage float64
+	lastOrder      string
+	timeToClose    float64
+	stock          string
+}
+
+var alpacaClient alpacaClientContainer
+
+func init() {
+	os.Setenv(common.EnvApiKeyID, "PKM4OPMZT6GJFCLF9ZCB")
```

Comment:
These are just paper keys, but you should probably at least regenerate them and remove them from public stuff like this.